Reqnroll/TestRunner.cs

```diff
@@ -39,7 +39,7 @@ public async Task OnFeatureEndAsync()
 
         public void OnScenarioInitialize(ScenarioInfo scenarioInfo, RuleInfo ruleInfo)
```

Comment:
I guess this method and the invokes from the generated code behind is still to be transformed to async.

Reply:
Should now be correctly transformed.